### PR TITLE
feat(testing): add unit tests for constants

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR refactors a few lines of code in `src/node/constants.ts` and adds a few unit tests to bring code coverage for the file to 100%.

## Changes
- refactor `constants` and add `getPackageJson` function to make it easier to test
- add `@schemastore/package` to devDeps to get PackageJson type
- add unit tests for constants 

## Screenshot
image

It increases our code coverage (Lines) from 49.53% to 50.09% (↑ 0.56%).
🟩 100% Line coverage (last 100) for `src/node/constants.ts`
image

TODOS
- [x] update screenshot
- [x] update percentages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,8 +63,8 @@
     "stylelint": "^13.0.0",
     "stylelint-config-recommended": "^3.0.0",
     "ts-node": "^9.0.0",
-    "wtfnode": "^0.8.4",
-    "typescript": "^4.1.3"
+    "typescript": "^4.1.3",
+    "wtfnode": "^0.8.4"
```
Comment (oxy) [inline package.json:67]: I wonder if this was a manual change or something done automatically? Not particularly an issue, just curious :eyes:

File: test/util.test.ts [hunk 2]
```diff
@@ -125,10 +125,6 @@ describe("util", () => {
   })
 
   describe("getOptions", () => {
-    // Things to mock
-    // logger
-    // location
-    // document
```
Comment (oxy) [inline test/util.test.ts:131]: Comment disappeared! no other changes in this file; curious about the change

**Review by oxy — COMMENTED** [review]
Looks good! Just curious about the disappearing comment, mostly.

File: test/util.test.ts (hunk 2 above)
Comment (jsjoeio) [reply test/util.test.ts:131]: That was intentional! See commit: https://github.com/cdr/code-server/pull/2701/commits/164d11e0277c8c39171c0b5398461a67bdb3ac72

File: package.json (hunk 1 above)
Comment (jsjoeio) [reply package.json:67]: I think that happens when you add a new dependency to the top? Not sure, I did see it in one of Asher's PRs

**Review by oxy — APPROVED** [review]
LGTM!

File: package.json (hunk 1 above)
Comment (code-asher) [reply package.json:67]: Yeah `yarn` resorts the dependencies. :smile:

Comment (code-asher) [reply package.json:67]: The real question is how it got out of order in the first place lol

Comment (jsjoeio) [reply package.json:67]: LOL I have no idea - could have been me 😂

Comment (code-asher) [reply package.json:67]: Haha idk I say we just blame `yarn`.

**Pull request merged**